lib/src/constants.dart

```diff
@@ -37,9 +47,16 @@ reporting=1
 # a number representing the version of the message that was
 # displayed.''';
 
+/// Name of the directory where all of the files necessary
+/// for this package will be located
+const String kDartToolDirectoryName = '.dart-tool';
+
 /// The minimum length for a session
 const int kSessionDurationMinutes = 30;
 
+/// Name for the json file where the session details will be stored
+const String kSessionFileName = 'session.json';
```

Comment:
can we namespace this, like `dash-analytics-session.json`, or something similar?